lib/tests/fixtures/sys/bios_vendor

```diff
@@ -0,0 +1 @@
+innotek GmbH
```

Comment:
I think t would be good to move these fixtures into a subdirectory or two, so it's clear that they're the "dmi virtualbox" sys files.